SolidStateGroup/bullet-train-api
# Feat/add-specific-permission-to-view-audit-logs

Thanks for submitting a PR! Please check the boxes below:


## Changes

Front change changes to allow the new VIEW_AUDIT_LOG permission for viewing audit logs. Also, moved the new permission from organisation level to project level.

## How did you test this code?

Manual testing:
- Assign the new VIEW_AUDIT_LOG permission to a common user (non-admin)
- Log in to the system as that user
- In the left menu see the option to sse audit logs and click on it.
- You can see the audit logs
